### DISCUSSION TITLE
【部品購入希望】「小頭ネジ M1.7」2本を購入できますか？

### DISCUSSION THREAD

**Discussion by KlingonPerson** [Yowkees/keyball#680] [General]
部品を紛失したため、上記部品を購入したいです。よろしくおねがいします

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
了解しました。お送り先等をご連絡したいので、お手数ですが下記リンクより必要部品をお知らせください。
https://shirogane-lab.net/inquiry
以上、よろしくお願いいたします。
